src/poetry/installation/wheel_installer.py

```diff
@@ -85,18 +74,13 @@ def __init__(self, env: Env) -> None:
                 script_kind = "win-arm64" if sys.maxsize > 2**32 else "win-arm"
             else:
                 script_kind = "win-amd64" if sys.maxsize > 2**32 else "win-ia32"
+        self.script_kind = script_kind
```

Comment:
```suggestion
        self._script_kind = script_kind
```
to make the interface clearer (same for `bytecode_optimization_levels`)